cypress/integration/customizedResources/user/add/AddRecord.spec.js

```diff
@@ -11,22 +11,21 @@ import { getFormValues,
 } from '../../../../support/helpersMethods';
   
 const { inputs, buttons, boardView } = customized;
-const { inputsTexts  } = common;
-  
+let email = 'aaaaaa' + faker.internet.email().toLowerCase();
```

Comment:
is that `toLowerCase()` necessary?

Reply:
Necessary, Admin automatically makes email to lower cases, so I have to do this in the test also. Without it, the test fails.
e.g.
'aaaaaasigurd2@gmail.com' to deeply equal 'aaaaaaSigurd2@gmail.com'  -Assertion failed